CONTRIBUTING.md

```diff
@@ -200,6 +200,16 @@ public string SomeProperty { get; set; }
 
 For general information about samples, see the [Samples Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-samples)
 
+## Updating Source on Build
+You can run `eng\scripts\Export-API.ps1` and `eng\scripts\Update-Snippets.ps1` simultaneously as part of the build by setting as true either:
+1. The property `UpdateSourceOnBuild` 
+2. The Environment variable `AZURE_DEV_UPDATESOURCESONBUILD`
```

Comment:
I would suggest changing this so folks know to test it to true `AZURE_DEV_UPDATESOURCESONBUILD=true`